src/utils/chart.ts

```diff
@@ -1,26 +1,35 @@
 import { HistoryData } from '../services/currencyService'
 
-export function setChart(labels: string[], values: number[], translatedLabel: string) {
+type ChartDataConfig = {
+  labels: string[]
+  datasets: ChartDataset[]
+}
+
+type ChartDataset = {
+  label: string
+  data: number[]
+  backgroundColor: string
+}
+
+export const setChart = (labels: string[], values: number[], translatedLabel: string): ChartDataConfig => {
+  const chartBackgroundColor = 'rgba(75,192,192,0.4)'
```

Comment:
Shouldn't we move that to const or something like that in another file ?

Reply:
Don't really know where to move it as this will be used only in chart config inside this single function. 

Additionally it will be changed in other task (changing chart bg) and don't know if we will be still using single `rgba()` value - maybe we would use some gradient and multiple colors.

Reply:
What do you think bro?

Reply:
Co myślisz człowieku?

Reply:
Mogę resolvować czy mam przenosić? jeśli przenosić to gdzie bro?

Reply:
Zostaw go gosciu bo jak Ty nie wiesz to ja CI przeciez nie bede niczego podawal :D

Reply:
lol co za Łeb 

![image](https://user-images.githubusercontent.com/1492786/107353158-046ca100-6acd-11eb-95fe-b4cb73ee25ec.png)